src/Components/Search/SearchBar.tsx

```diff
@@ -253,11 +256,10 @@ export class SearchBar extends Component<Props, State> {
     { containerProps, children, query },
     { xs }
   ) => {
+    const noResults = get(this.props, p => p.viewer.search.edges.length === 0)
     const { focused } = this.state
-    if (!focused) {
-      return null
-    }
-    if (xs && !query) {
+
+    if (noResults || !focused || !query || (xs && !query)) {
```

Comment:
Fixes shadow issue in search bar when focused and content is empty: 

<img width="208" alt="Screen Shot 2019-06-04 at 6 43 11 PM" src="https://user-images.githubusercontent.com/236943/58924426-a634cb80-86f8-11e9-8a30-1152d1688773.png">

Reply:
The condition `(xs && !query)` is covered by the condition `!query`, and could be removed.

Reply:
Will do